style.css

```diff
@@ -198,6 +198,183 @@ body {
   z-index: 2;
 }
 
+.order {
+  background: #FAE8E4;
+  padding-left: 137px;
+  width: 100%;
+}
+
+.order-images {
+  display: flex;
+  width: 50%;
+}
+
+.order-image__first {
+  transform: translateY(-28px);
+  z-index: 1;
+}
+
+.order-image__second {
+  margin-top: 27px;
+  transform: translateX(-303px);
+  z-index: 2;
+}
+
+.order-content {
+  background: #FFFFFF;
```

Comment:
Codacy found an issue: [Expected "#FFFFFF" to be "#FFF" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)